libqtile/widget/volume.py

```diff
@@ -67,6 +68,12 @@ class Volume(base._TextBox):
             "Use emoji to display volume states, only if ``theme_path`` is not set."
             "The specified font needs to contain the correct unicode characters.",
         ),
+        (
+            "emoji_list",
+            ["\U0001f507", "\U0001f508", "\U0001f509", "\U0001f50a"],
+            "List of emojis/font-symbols to display volume states, only if ``emoji`` is set."
+            "List contains 4 symbols, from lowest volume to highest.",
```

Comment:
The concatenated string of these two lines needs a space:
```suggestion
            " List contains 4 symbols, from lowest volume to highest.",
```

Though I notice that the options `emoji` and `step` also have this issue: http://docs.qtile.org/en/stable/manual/ref/widgets.html#libqtile.widget.Volume. Maybe you could put spaces on those lines too while you're there? :)